src/pages/index.js

```diff
@@ -168,6 +200,7 @@ function handleProfileEditSubmit({ title, description }) {
       userInfo.setUserInfo(data.name, data.about);
       editFormValidator.disableSubmitButton();
       editProfilePopup.close();
+      editProfilePopup.renderLoading(false);
```

Comment:
`.renderLoading(false);` should be in `finally`

You need to return the **default** button text only in blocks `finally` so that the button could get the text back in any case: `success` or `failure`.

**Please, correct it everywhere**

Reply:
makes sense. thank you